src/Components/Forms/FilterForm.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+
+export default function FilterForm({characterName, setCharacterName}) {
+    const charNameArr = ['Michael', 'Pam', 'Jim','Andy','Angela','Charles','Creed', 'Darryl', 'David', 'Deangelo', 'Dwight', 'Ed', 'Erin', 'Gabe', 'Holly', 'Jan', 'Jo', 'Josh', 'Kelly', 'Kevin', 'Oscar', 'Phyllis', 'Ryan', 'Stanley', 'Toby', 'Todd']
```

Comment:
it would be cool to get this list from the API if it has that functionality (I'm assuming it didn't)